lsp-completion.el

```diff
@@ -525,7 +525,15 @@ The MARKERS and PREFIX value will be attached to each candidate."
        :company-require-match 'never
        :company-prefix-length
        (save-excursion
-         (and (lsp-completion--looking-back-trigger-characterp trigger-chars) t))
+         (defvar company-minimum-prefix-length)
```

Comment:
I rarely see something like this. Why not put it in the root level? 🤔

Reply:
It would be the same, since I only use the `company-minimum-prefix-length` here, I keep this in this functin for cleanliness. Functionality wise, this is just forward declaration and should be okay to put anywhere before its usage.